AUTHORS.txt

```diff
@@ -18,6 +24,7 @@ The SeisSol group appreciates the work of following contributors and former core
  * Jordan, Kirk - IBM Research
  * Kaeser, Martin - MunichRE
  * Krenz, Lukas - TUM
+ * Kurapati, Vikas - TUM
```

Comment:
this list is listing `former core group members`

Reply:
It says `contributors and former core members`, doesn't that mean current contributors and former core members? Or do we mean "former contributors and core members"?

Reply:
Ok; removed myself and Vikas then. :)
For now

Reply:
you are include in `The SeisSol group`